styles/go.dev/catppuccin.user.css

```diff
@@ -279,10 +295,6 @@
       }
     }
     // Google Logo SVG for both go.dev and pkg.go
```

Comment:
Did you mean to remove this comment as well?

Reply:
Ah yes, good spot.